packages/2d/src/Hooks/useInspectorSelectEntity.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity } from "@hex-engine/core";
+import { useInspectorSelect } from "@hex-engine/inspector";
+
+export default function useInspectorSelectEntity(getEntity: () => Entity) {
+  if (process.env.NODE_ENV === "production") return;
+
+  const { getSelectMode, inspectEntity } = useInspectorSelect();
+  const entity = getEntity();
```

Comment:
Could we move the `getEntity` call inside of the `if(getSelectMode())` conditional body?

Reply:
Yes, on the other hand I'm not even sure I need the `getEntity` function. I just added it to match code style with `  useInspectorHoverOutline(() => geometry.shape);`. Is there a reason you pass a function instead of `geometry.shape` directly?

Reply:
I moved it into the if.